scripts/md2rst.py

```diff
@@ -27,7 +27,9 @@ def download_pandoc():
 
 
 def md2rst(text):
+    markdown_flavour = os.environ.get('FORMAT', 'markdown_github')
```

Comment:
I think you misunderstood my previous suggestion. I was talking about the environment variable named 'FORMAT'. please rename that.